README.md

```diff
@@ -1,86 +1,46 @@
-AntidoteDB
-============
+![Vaxine logo](https://vaxine.io/id/vaxine-logo-dark.png#gh-dark-mode-only)
+![Vaxine logo](https://vaxine.io/id/vaxine-logo-light.png#gh-light-mode-only)
```

Comment:
FYI this doesn't seem to work on images not hosted on Github. Moreover, they now recommend [a different way](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)...